library/azure_rm_devtestlabslab.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/python
```

Comment:
about file name `azure_rm_devtestlabslab.py`,  seems there's no `s` in existing azure module names, suggest to `azure_rm_devtestlablab.py`